standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -240,7 +239,9 @@ private void startLocalSegmentStore(int segmentStoreId) throws Exception {
                                           .with(ServiceConfig.THREAD_POOL_SIZE, THREADPOOL_SIZE)
                                           .with(ServiceConfig.ZK_URL, "localhost:" + zkPort)
                                           .with(ServiceConfig.LISTENING_PORT, this.segmentStorePorts[segmentStoreId])
-                                          .with(ServiceConfig.CLUSTER_NAME, this.clusterName))
+                                          .with(ServiceConfig.CLUSTER_NAME, this.clusterName)
+                                          .with(ServiceConfig.STORAGE_IMPLEMENTATION, isInMemStorage ?
```

Comment:
Please make these some constants somewhere. If you need to reuse the same string(s) in more than one place, then refer to them somewhere easily accessible.